Monika After Story/game/event-rules.rpy

```diff
@@ -714,6 +715,63 @@ init -1 python:
             return ev.rules.get(EV_RULE_PRIORITY, MASPriorityRule.DEF_PRIORITY)
 
 
+    class MASProbabilityRule(object):
+        """
+        Static class used to create probability rules.
+
+        Priority rules are just integers that determine the probability of something being selected.

```

Comment:
```suggestion
        Probability rules are just integers that determine the probability of something being selected.

```

Assuming this is what you mean.

Can you also give an example in this comment of what it means to use this? Like if you pick 6, how does that affect the selection process? Is this 6 times more likely to be chosen? Does this set the weighting of this selection as if this item was copied 6 times into a list?